sdk/keyvault/azure-security-keyvault/azure/security/keyvault/keys/_client.py

```diff
@@ -3,264 +3,610 @@
 # Licensed under the MIT License. See LICENSE.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
-import functools
-from typing import Any, List, Mapping, Optional
-import uuid
-
-from ._models import (
-    DeletedKey,
-    DeletedKeyItem,
-    DeletedKeyItemPaged,
-    JsonWebKey,
-    Key,
-    KeyAttributes,
-    KeyCreateParameters,
-    KeyItem,
-    KeyItemPaged,
-    KeyUpdateParameters,
-)
-from azure.core.configuration import Configuration
-from azure.core.exceptions import HttpResponseError
-from azure.core.pipeline import Pipeline
-from azure.core.pipeline.policies import (
-    HTTPPolicy,
-    UserAgentPolicy,
-    HeadersPolicy,
-    RetryPolicy,
-    RedirectPolicy,
-    ContentDecodePolicy,
-)
-from azure.core.pipeline.transport import RequestsTransport, HttpRequest, HttpResponse, HttpTransport
-from msrest import Serializer, Deserializer
-
-
-class BearerTokenCredentialPolicy(HTTPPolicy):
-    def __init__(self, credentials):
-        self._credentials = credentials
+from typing import Any, Dict, Generator, Mapping, Optional, List
+from datetime import datetime
 
-    def send(self, request, **kwargs):
-        # type: (HttpRequest, Any) -> HttpResponse
-        auth_header = "Bearer " + self._credentials.token["access_token"]
-        request.http_request.headers["Authorization"] = auth_header
+from azure.core import Configuration
+from azure.core.exceptions import ResourceExistsError, ResourceNotFoundError
+from azure.core.pipeline import Pipeline
+from azure.core.pipeline.policies import RetryPolicy, RedirectPolicy, UserAgentPolicy
+from azure.core.pipeline.transport import RequestsTransport
+from azure.security.keyvault._internal import _BearerTokenCredentialPolicy
 
-        return self.next.send(request, **kwargs)
+from .._generated import KeyVaultClient
+from ._models import Key, KeyBase, DeletedKey
 
 
 class KeyClient:
-    API_VERSION = "7.0"
+    """KeyClient defines a high level interface for
+    managing keys in the specified vault.
+
+    :param credentials:  A credential or credential provider which can be used to authenticate to the vault,
+        a ValueError will be raised if the entity is not provided
+    :type credentials: azure.authentication.Credential or azure.authentication.CredentialProvider
+    :param str vault_url: The url of the vault to which the client will connect,
+        a ValueError will be raised if the entity is not provided
+    :param ~azure.core.configuration.Configuration config:  The configuration for the KeyClient
+
+    Example:
+        .. literalinclude:: ../tests/test_examples_keyvault.py
+            :start-after: [START create_key_client]
+            :end-before: [END create_key_client]
+            :language: python
+            :dedent: 4
+            :caption: Creates a new instance of the Key client
+    """
+
+    _api_version = "7.0"
 
     @staticmethod
     def create_config(**kwargs):
```

Comment:
I'd give this an empty body and a TODO because it isn't useful as is and a good replacement requires changes elsewhere.